src/server/server.ts

```diff
@@ -138,7 +141,7 @@ export class GraphQLTransportWSServer {
           if (this.onConnect) {
             onConnectPromise = new Promise((resolve, reject) => {
               try {
-                resolve(this.onConnect(parsedMessage.payload, connection));
+                resolve(this.onConnect(parsedMessage.payload, connectionContext));
```

Comment:
hmm,
we need to update documenation that socket is now aviable on Connect, so there are more info a connection handler can get. (upgrade url for example)